### PR TITLE
Update CI config

### LINKED ISSUE #14: Update CI config
CI config should updated for better maintenance on dependencies and a simpler usage

### PR DESCRIPTION
**Pull request by Mael-Le-Garrec**
Fixes #14

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,9 +1,6 @@
 [bumpversion]
 current_version = 0.1.3
 commit = True
-tag = True
-
-[bumpversion:file:setup.py]
```
Comment (lmalina) [inline .bumpversion.cfg]: doc/conf.py?

File: setup.py [hunk 2]
```diff
@@ -22,7 +59,11 @@
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```
Comment (lmalina) [inline setup.py]: Add "Programming Language :: Python :: 3.8",

File: .bumpversion.cfg (hunk 1 above)
Comment (Mael-Le-Garrec) [reply .bumpversion.cfg]: What about it? It's below, github just decided to hide it and you need to expand

Comment (lmalina) [reply .bumpversion.cfg]: yes, sorry

**Pull request merged**